app/src/org/commcare/activities/FormEntryActivityUIController.java

```diff
@@ -562,6 +573,10 @@ protected void next() {
         }
     }
 
+    protected boolean shouldIgnoreNavigationAction() {
```

Comment:
No, they're not. `shouldIgnoreSwipeAction` is used in the way its name indicates; it only acts to prevent *swipe* actions. I don't know why this is and chose not to mess with it